Tchap/Modules/Room/Members/RoomParticipantsViewController.m

```diff
@@ -327,10 +330,12 @@ - (void)viewDidLayoutSubviews
         {
             newBounds.origin = CGPointZero;
             
+            MXWeakify(self);
```

Comment:
weak self strong self dance is not necessary here. Self is retained during the animation duration only.